src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,79 +60,173 @@ public RobotContainer() {
+      );
+
     new Trigger(() -> m_driverController.getYButton())
-      .onTrue(new VisionTranslateCommand(m_vision, m_robotDrive, m_driverController));
+      .onTrue(
+        new VisionTranslateCommand(m_vision, m_robotDrive, m_driverController)
+      );
 
     new Trigger(() -> triggerPressed())
       .whileTrue(new SimpleDriveCommand(m_robotDrive, m_driverController));
 
-    new Trigger(() -> m_operatorController.getRawButton(1))
-      .whileTrue(new ReleaseCommand(m_intake, m_intake.getWAxisSpeedMultiplier(OIConstants.INTAKE_SPEED_AXIS)));
-
-    new Trigger(() -> m_operatorController.getRawButton(2))
-      .whileTrue(new IntakeCommand(m_intake, m_intake.getWAxisSpeedMultiplier(OIConstants.INTAKE_SPEED_AXIS)));
-
-    //stops the intake motors and holds the current piece when button 14 is pressed
-    new Trigger(() -> m_operatorController.getRawButton(14))
-      .onTrue(new InstantCommand(() -> m_intake.stopMotors(), m_intake));
-
-    //sets the current game pice type to cones when button 4 is pressed
-    new Trigger(() -> m_operatorController.getRawButton(4))
-      .onTrue(new InstantCommand(() -> m_intake.setState(IntakeGamePiece.CONE), m_intake));
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.releaseGamePiece))
+      .whileTrue(
+        new ReleaseCommand(
+          m_intake,
+          m_intake.getWAxisSpeedMultiplier(OIConstants.INTAKE_SPEED_AXIS)
+        )
+      );
+
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.intakeGamePiece))
+      .whileTrue(
+        new IntakeCommand(
+          m_intake,
+          m_intake.getWAxisSpeedMultiplier(OIConstants.INTAKE_SPEED_AXIS)
+        )
+      );
+
+    //sets the current game piece type to cones when button 4 is pressed
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.setGamePieceCone))
+      .onTrue(
+        new InstantCommand(
+          () -> m_intake.setState(IntakeGamePiece.CONE),
+          m_intake
+        )
+      );
 
     //sets the current game piece type to cubes when button 3 is pressed
-    new Trigger(() -> m_operatorController.getRawButton(3))
-      .onTrue(new InstantCommand(() -> m_intake.setState(IntakeGamePiece.CUBE), m_intake));
-    
-    //fully lowers the arm when button 16 is pressed
-    new Trigger(() -> m_operatorController.getRawButton(16))
+    new Trigger(() -> m_operatorController.getRawButton(Bindings.setGamePieceCube))
+      .onTrue(
+        new InstantCommand(
+          () -> m_intake.setState(IntakeGamePiece.CUBE),
+          m_intake
+        )
+      );
+
+    new Trigger(() -> m_operatorController.getRawButton(m_operatorController.getPOV(0)))
       .onTrue(new InstantCommand(m_arm::retract));
-       
-    //fully raises the arm when button 15 is pressed
-    new Trigger(() -> m_operatorController.getRawButton(15))
+
+    new Trigger(() -> m_operatorController.getRawButton(m_operatorController.getPOV(180)))
```

Comment:
It'd be nice if we had these comments still... It makes it easier to read the code at a glance (rather than having to parse through the individual lines)